doc/plugin_agent_workloadattestor_docker.md

```diff
@@ -0,0 +1,61 @@
+# Agent plugin: WorkloadAttestor "docker"
+
+The `docker` plugin generates selectors based on docker labels for workloads calling the agent.
+It does so by retrieving the workload's container ID from its cgroup membership, then querying
+the docker daemon for the container's labels.
+
+| Configuration | Description |
+| ------------- | ----------- |
+| docker_socket_path | The location of the docker daemon socket (default: "unix:///var/run/docker.sock" on unix). |
+| docker_version | The API version of the docker daemon (default: "1.25").
+| cgroup_prefix | The cgroup prefix to look for in the cgroup entries (e.g. "/docker"). |
+| cgroup_container_index | The index within the cgroup path where the container ID should be found (default: 0). |
+
+Since selectors are created dynamically based on the container's docker labels, there isn't a list of known selectors.
+Instead, all of the container's labels are used in creating the list of selectors.
+
+| Selector | Value |
+| -------- | ----- |
+| docker:label:abc | The value of the docker label `abc` |
+| docker:label:xyz | The value of the docker label `xyz` |
```

Comment:
nit: I know that this is taken from the k8s workload attestor docs, but I think separating by selector and value is confusing to users. Even though the difference between type and value is distinguished internally, we don't really surface that very strongly in the CLI tools. I'd suggest a table like this one instead: https://github.com/spiffe/spire/blob/master/doc/plugin_server_nodeattestor_k8s_sat.md